### ISSUE TITLE
Bug uploading picture into alert

### ISSUE THREAD

**Issue by priamai** [StrangeBeeCorp/TheHive-feedback#5]
### Request Type
Bug

### Work Environment

| Question                   | Answer                                |
| -------------------------- | ------------------------------------- |
| OS version (server)        | Ubuntu  |
| OS version (client)        | Windows |
| Virtualized  Env.          | False                          |
| Dedicated RAM              | 32 GB                                 |
| vCPU                       |  8       |
| TheHive version            | 5.x.x                                 |
| Package Type               | Docker |
| Database                   | Builtin               |
| Index type                 | Builtin              |
| Attachments storage        | Local           |
| Browser type & version     | Chrome         |

### Problem Description
In an incident I try to upload an image.
Nothing happens and lot of red errors in the browser console.

### Steps to Reproduce
1. Create a case from an alert
2. Go to Pages
3. Create a Page
4. Add an image

### Complementary information

![image](https://user-images.githubusercontent.com/57333254/171562339-4eca3a44-3fdb-4425-802f-12a42f294649.png)

My docker setup:

```
version: "3"
services:
  thehive:
    image: strangebee/thehive:latest
    mem_limit: 1500m
    user: root
    ports:
      - "9000:9000"
    environment:
      - JVM_OPTS="-Xms1024M -Xmx1024M"
    command:
      - --secret
      - "victoriasecret2022!"
      - "--no-config-cortex"
    volumes:
      - thehive_db:/data/db
      - thehive_files:/data/files
      - thehive_index:/data/index
volumes:
  thehive_db:
  thehive_files:
  thehive_index:

```

**Comment by vdebergue** [comment]
It seems like there is an issue with the docker entrypoint where it's trying to configure S3 for file storage instead of the local folder.
Adding this issue to 5.0.8

The only way around this at the moment is to use a custom config or to use a minio container

**Comment by priamai** [comment]
Understood, I will add a Minio service to my docker composer file.
Cheers.

**Comment by vdebergue** [comment]
TheHive 5.0.8 was releases with the fix: https://docs.strangebee.com/thehive/release-notes/release-notes-5.0/#508-15-june-2022

Please reopen the issue if you have more issues with that or other questions. Thanks.

**Comment by vdebergue** [comment]
Oops just saw that the issue is still there with 5.0.8 (bash is hard), reopening it and fixing it for next release.

It will work correctly by providing the arguments `--storage-directory <path_to_storage>`

**Comment by vdebergue** [comment]
This is fixed in 5.0.9
